src/xcode/ENA/TestPlans/Screenshots.xctestplan

```diff
@@ -44,6 +44,8 @@
         "ENAUITests_01b_Statistics\/test_screenshot_statistics_card_titles()",
         "ENAUITests_03_Settings\/test_0032_SettingsNotificationsOn()",
         "ENAUITests_03_Settings\/test_0033_SettingsNotificationsOff()",
```

Comment:
you need to explicitly delete the renamed tests from this list.

Reply:
Good catch, I removed them